### ISSUE TITLE
Add Power-ups to the game at random regions

### ISSUE THREAD

**Issue by VRamazing** [igameproject/line-ball#3]

**Comment by Jake0Tron** [comment]
Will give this a shot, what kind of powerups did you have in mind? I'll start on a free life...

**Comment by VRamazing** [comment]
Yeah I was thinking about that too for starters.Make it square and color that is easy to catch eye

**Comment by Jake0Tron** [comment]
sure thing.

**Comment by Jake0Tron** [comment]
see #8

**Comment by Rogue00** [comment]
I'm interested in helping with this issue but I'm not entirely sure if my skill in javascript is up to snuff. The first thing I see is that your game is scrolling by much too fast. Also, your ball up and down speed (accelerate using the space bar and gravity) is also set too high (fast) for this game. It's impossible to play at this stage.

**Comment by Jake0Tron** [comment]
I believe this is resolved?

**Comment by VRamazing** [comment]
Maybe We can slowly increase the speed of ball with time.I was thinking about moving the ball only after the user presses the spacebar once

**Comment by VRamazing** [comment]
@Jake0Tron  Have to go through your PR.Had been a bit busy.Will do by tomorrow.

**Comment by cedric-h** [comment]
Since he said that he'd do this tomorrow and it's been eight days since then, is this up-for-grabs?

**Comment by Jake0Tron** [comment]
I mean, re-writing the code won't make the PR go any differently if its not happening yet, so... I dunno @DefecateRainbows , we'll let @VRamazing decide that part
